modules/installation-configuration-parameters.adoc

```diff
@@ -1246,6 +1246,18 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 |The GCP service account used for the encryption request for control plane and compute machines. If absent, the Compute Engine default service account is used. For more information about GCP service accounts, see Google's documentation on link:https://cloud.google.com/compute/docs/access/service-accounts#compute_engine_service_account[service accounts].
 |The GCP service account email, for example `<service_account_name>@<project_id>.iam.gserviceaccount.com`.
 
+|`platform.gcp.defaultMachinePlatform.secureBoot`
+|Whether to enable Shielded VMs for all machines in the cluster. Shielded VMs have additional security protocols such as secure boot, firmware and integrity monitoring, and rootkit protection. For more information on Shielded VMs, see Google's documentation on link:https://cloud.google.com/shielded-vm[Shielded VMs].
```

Comment:
This option is specific to secure boot which is just one of the shielded VMs options. 
Consider:
```suggestion
|Whether to enable Shielded VMs secure boot for all machines in the cluster. Shielded VMs have additional security protocols such as secure boot, firmware and integrity monitoring, and rootkit protection. For more information on Shielded VMs, see Google's documentation on link:https://cloud.google.com/shielded-vm[Shielded VMs].
```